xgcm/test/test_metrics.py

```diff
@@ -163,3 +163,16 @@ def test_set_metric():
 
         for metric_expected, metric in zip(v, grid._metrics[k]):
             xr.testing.assert_equal(metric_expected.reset_coords(drop=True), metric)
+
```

Comment:
I think we should also have a test for `interp_like`, but I think that should live in `test_grid.py`.